.travis.yml

```diff
@@ -1,4 +1,4 @@
-language: generic
+language: c
```

Comment:
Yay!

Do we want to specify compiler?  (Or have a matrix?)

For example:

```
compiler:
  - clang
  - gcc
```
(From: https://docs.travis-ci.com/user/languages/c)